source/common/stats/utility.cc

```diff
@@ -16,9 +17,16 @@ std::string Utility::sanitizeStatsName(absl::string_view name) {
   if (absl::StartsWith(name, ".")) {
     name.remove_prefix(1);
   }
+
   std::string stats_name = std::string(name);
-  std::replace(stats_name.begin(), stats_name.end(), ':', '_');
-  std::replace(stats_name.begin(), stats_name.end(), '\0', '_');
+  absl::StrReplaceAll(
+      {
+          {":", "_"},
+          {"\0", "_"},
+          {"_//", "_"},
+          {"_/", "_"},
+      },
+      &stats_name);
```

Comment:
these changes are OK with me but can you check CI and add unit tests for the new cases?

wondering also if performance might be an issue as we are now iterating over the string 4x. This gets called for every stat created in the system.

It's probably OK for now compared with the other work that goes on per stat, but if this grows much further we should consider writing a single loop with a switch statement.

Reply:
Sure, I will add UT and fix CI.

I find some functions to do substring replace, not sure this one is the most performant, which function do you recommend? Or do you prefer "check condition" then "replace" to reduce iterating times?

Reply:
I wouldn't mess with it yet; if the number of substitutions grows we should try to make the swap with a single iteration over the characters in the string.

In that case I wouldn't use a function ; something more like 
```
for (uint32_t i = 0, n = in.size(); i < n; ++i) {
  char c = in[i];
  switch (c) {
    case ':':
      if (in.substr(i + 1, 3) == "://") {
        i += 2;
      } else if (in.substr(i + 1, 2) == ":/")) {
        ++i;
      }
      // fall through
    case '\0':
      out += "_";
      break;
   default:
      out += c;
      break;
  }
  return out;
}
```
something like that. It's a lot harder to read than what you have and would require a lot of testing, so until we see what you have is too slow I'd leave it alone, in favor of the more readable code you have.

Reply:
Agree that it is harder to read and maintain, but performant since only one loop.

Reply:
I do not find specific UT for `Utility::sanitizeStatsName`, so I create a new one.